### DISCUSSION TITLE
autosave files

### DISCUSSION THREAD

**Discussion by dim2r** [WinMerge/winmerge#1796] [Ideas]
I often perform the following task --- 
1) load text from database into temporary file
2) compare temporary text with some file at the disk
3) load merged result into database

When i run winmerge i need the following functionality
1) Save file immediately after i moved any changes from one file into another. 
2) Fast exit winmerge by one click or keyboard shortcut
This behavior would save much time for me because i often perform such tasks.

So i propose to add some command line switches to turn on that behavior.
1) --quick_save
2) --quick_exit

I think that is not a big deal to implement such switched and they would be usefull for other people.

Best wishes ,
Dmitry

**Comment by DavidHollman** [comment]
Another use of a feature like this would be when WinMerge is run for 3-way merges from another tool. I have it setup with the /am option for automerge, but this still requires me to click OK to the "no conflicts..." messagebox and then save the middle file manually.  An autosave option in combination with /am would save some or all of those steps.
